ci-operator/config/openstack-k8s-operators/neutron-operator/openstack-k8s-operators-neutron-operator-master.yaml

```diff
@@ -76,6 +81,19 @@ tests:
   optional: true
   steps:
     workflow: openstack-k8s-operators-test-build-deploy
+- as: precommit-check
+  commands: |
+    export SKIP=no-commit-to-branch
+    export GOFLAG=
+    export PRE_COMMIT_HOME=/tmp
+    export GOLANGCI_LINT_CACHE=/tmp
+    mkdir -p ../operator/ && cp -r . ../operator
+    cd ../operator
+    pre-commit run -a --show-diff-on-failure
+  container:
+    clone: true
```

Comment:
@viroel do we need `clone: true` here , I don't see it in `openstack-operator`, it was removed with PR [2] so I want to cross-check if that was intentional.

https://github.com/openshift/release/blob/master/ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml#L102-L103
[2] https://github.com/openshift/release/pull/36038/files

Reply:
Answering myself:

If we build the container from src we already get the cloned repo, but if we use a different container we need `clone:true` , As we are using `src`  to build` openstack-operator-ci-pre-commi`t we don't need `clone: true here.

Reply:
Removing `clone:true`